libmproxy/script.py

```diff
@@ -192,3 +195,23 @@ def _concurrent(ctx, obj):
         return _concurrent
     raise NotImplementedError(
         "Concurrent decorator not supported for '%s' method." % fn.func_name)
+
+
+class ScriptModified(PatternMatchingEventHandler):
+    
+    def __init__(self, flow_master):
+        self.flow_master = flow_master
+        PatternMatchingEventHandler.__init__(self, ignore_directories=True, patterns=["*.py"])
+        self.context = ScriptContext(self.flow_master)
+
+    def on_modified(self, event=FileModifiedEvent):
+        self.flow_master.reload_scripts()
```

Comment:
2. Instead of using the flow master here, trigger the blinker event with the script object as argument.